src/tasks/copy-template-files.ts

```diff
@@ -189,6 +256,32 @@ const processTemplatedFiles = async (
         templateTargetName
       );
       fs.writeFileSync(targetPath, output);
+
+      if (isDev) {
+        const hasCombinedArgs = Object.keys(combinedArgs).length > 0
+        const hasArgsPaths = argsFileUrls.length > 0
+        const devOutput = `--- TEMPLATE FILE
+templates/${templateFileDescriptor.source}${templateFileDescriptor.relativePath}
+
+
+--- ARGS FILES
+${hasArgsPaths
+  ? argsFileUrls.map(url => `\t- ${url.split('scaffold-eth-2/')[1]}`).join('\n')
```

Comment:
> Regarding "ARGS FILES" section, I just created the dev mode projects with hardhat, foundry, and no framework. I'm not getting undefined. My results:

I think I found the bug since we are using here: "scaffold-eth-2/"  to split and I don't  it name "scaffold-eth-2" locally its named "cli" so it giving me undefined 

I tried logging the whole path and check this out : 
![Screenshot 2023-08-10 at 9 15 26 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/460c153e-21bd-4a4f-9c53-756d492c7a43)


Here is my propose solution : 

```ts
argsFileUrls.map((url) => `\t- ${path.join("templates", url.split("templates")[1])}`).join("\n")

```

Splitting on the basis of `"templates`" because irrespective of how people name it locally we will have "templates" always in the path since `.args.mjs` will only be present in "templates"


But feel free update the logic 🙌

Reply:
oh that makes sense. I like your proposal. I'll make a commit as soon as I find the time.

Thanks!

Reply:
Fixed!